packages/paste-website/src/components/shortcodes/generic-header/index.tsx

```diff
@@ -19,8 +20,8 @@ const IconAnchor: React.FC<React.PropsWithChildren<{href: string; icon: React.Re
   children,
 }) => (
   <Anchor href={href}>
-    <Box as="span">
-      <Box as="span" display="inline-block" marginRight="space20">
+    <Box as="span" display="flex" alignItems="center" columnGap="space20">
```

Comment:
Note: this is going to get blown away by the new Sidebar Simon is merging. Needed this to properly position an external link icon near some library pages in the sidebar